python/graphstorm/model/rgcn_encoder.py

```diff
@@ -50,61 +50,53 @@ class RelGraphConvLayer(nn.Module):
-
     Note:
-    -----
-    * In the RelGraphConvLayer, the implementation select 'right' or the default option for the norm
-    to divide the aggregated messages by each node's in-degrees, which is equivalent to averaging
-    the received messages.
+    ******
+    * The implementation of ``RelGraphConvLayer`` selects `right` as the norm, which divides
+      the aggregated messages by each node's in-degrees, equivalent to averaging the received
+      messages.
 
     Examples:
     ----------
 
     .. code:: python
 
         # suppose graph and input_feature are ready
-        from graphstorm.model.rgcn_encoder import RelGraphConvLayer
+        from graphstorm.model import RelGraphConvLayer
 
         layer = RelGraphConvLayer(
-                h_dim, h_dim, g.canonical_etypes,
-                num_bases, activation, self_loop,
+                in_feat=h_dim, out_feat=h_dim, rel_names=g.canonical_etypes,
+                num_bases=num_bases, self_loop,
                 dropout, num_ffn_layers_in_gnn,
                 ffn_activation, norm)
         h = layer(g, input_feature)
 
     Parameters
     ----------
-    in_feat : int
+    in_feat: int
         Input feature size.
-    out_feat : int
+    out_feat: int
         Output feature size.
-    rel_names : list[str]
-        Relation names.
-    num_bases : int
-        Number of bases. If is none, use number of relations. Default: None.
-    weight : bool, optional
-        True if a linear layer is applied after message passing. Default: True
-    bias : bool, optional
-        True if bias is added. Default: True
-    activation : callable, optional
-        Activation function. Default: None
-    self_loop : bool, optional
-        True to include self loop message. Default: False
-    dropout : float, optional
-        Dropout rate. Default: 0.0
-    num_ffn_layers_in_gnn: int, optional
-        Number of layers of ngnn between gnn layers
+    rel_names: list of tuple
+        Relation type list in the format of [('src_ntyp1', 'etype1', 'dst_ntype1`), ...].
+    num_bases: int
+        Number of bases. If is None, use number of relation types. Default: None.
+    weight: bool
+        Whether to apply a linear layer after message passing. Default: True.
+    bias: bool
+        Whether to add bias. Default: True.
+    activation: callable
+        Activation function. Default: None.
+    self_loop: bool
+        Whether to include self loop message. Default: False.
+    dropout: float
+        Dropout rate. Default: 0.
+    num_ffn_layers_in_gnn: int
+        Number of fnn layers between gnn layers. Default: 0.
     ffn_actication: torch.nn.functional
-        Activation Method for ngnn
-    norm : str, optional
-        Normalization Method. Default: None
+        Activation for ffn. Default: relu.
+    norm: str
+        Normalization methods. Options:``batch`` and ``layer``. Default: None.
```

Comment:
I think we should list None in the options, if we delete the optional in the text

Reply:
Agree. And we need to update all the files related.

Reply:
Updated to

`
Normalization methods. Options:``batch``, ``layer``, and ``None``. Default: None,  meaning not do normalization.
`